### PR TITLE
[DOCFIX] Add Fuse in worker doc

### PR DESCRIPTION
**Pull request by LuQQiu**

### CODE REVIEW

File: docs/en/api/POSIX-API.md [hunk 1]
```diff
@@ -140,6 +162,34 @@ pid mount_point alluxio_path
...
+
+All the information about the Fuse mount needs to be provided as the worker configuration.
+```
+alluxio.worker.fuse.mount.alluxio.path=<alluxio_path>
+alluxio.worker.fuse.mount.point=<mount_point>
+alluxio.worker.fuse.mount.options=<list of mount options separated by comma>
+```
+
+For example, the following example mounts Alluxio path `/mnt/people` to local path `/`
+with `kernel_cache,entry_timeout=7200,attr_timeout=N` mount options.
```
Comment (apc999) [inline docs/en/api/POSIX-API.md:189]: is it just `N` or you are indicating a number?

File: docs/en/api/POSIX-API.md [hunk 2]
```diff
@@ -140,6 +163,37 @@ pid mount_point alluxio_path
...
+
+```
+alluxio.worker.fuse.enabled=true
+```
+
+All the FUSE configuration needs to be provided as the worker configuration.
+
+```
+alluxio.worker.fuse.mount.alluxio.path=<alluxio_path>
+alluxio.worker.fuse.mount.point=<mount_point>
```
Comment (apc999) [inline docs/en/api/POSIX-API.md:184]: this is required,  right? then put it togetther with `alluxio.worker.fuse.enabled` above

**Comment by apc999** [comment]
@LuQQiu I made a pass and left some comments . PTAL

File: docs/en/api/POSIX-API.md (hunk 1 above)
Comment (LuQQiu) [reply docs/en/api/POSIX-API.md]: Changed to 7200, thanks!

File: docs/en/api/POSIX-API.md (hunk 2 above)
Comment (LuQQiu) [reply docs/en/api/POSIX-API.md]: Yeah, it's required, move it upward

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**